goal_x: -1.7
goal_y: 1.07
max_time: 120.0
distance_threshold: 0.8
